src/less/components/search/SearchFacade.less

```diff
@@ -0,0 +1,42 @@
+/*
+  Mixins to style <hx-search> facade states.
+*/
+
+.SearchFacade(pristine) {
+  background-color: @gray-0;
+  border-radius: 2px;
+  border: 1px solid @gray-500;
+  color: @gray-900;
+  font-family: inherit;
+  font-size: 1rem;
+}
+
+.SearchFacade(pristine-focus) {
+  border: 1px solid @cyan-700;
+  box-shadow: @focus-glow;
+  outline: 0;
```

Comment:
Is `outline` something we can extract into Light DOM styles?

Reply:
`outline` removed.